### ISSUE TITLE
Comparison Ads (Part 1) (Art Request)

### ISSUE THREAD

**Issue by andyvan-ph** [PostHog/meta#233]
As part of https://github.com/PostHog/meta/issues/223, I want to test running some comparative ads against Hotjar, Mixpanel, Amplitude and LaunchDarkly.

We're going to try a few different approaches here, including some that are more meme-y and fun, but also ones that are more info dense. This is about the latter.

What I'm looking for:

- A 1:1 aspect ad with a comparison table + space for an eye-catching headline / strapline / quote
- Minimum res 1,200 x 1,200
- Some kind of fun visual theme. I know we've explored a beat em' up / kungfu dojo type vibe in the past.
- Feedback from everyone on tables / copy ideas etc. The below is a starting point.
- I fully recognise some of the headlines are not great. Please share better ones!

## Hotjar

|                            | **Hotjar** | **PostHog** |
|---------------------------------|------------|-------------|
| 5k free recordings every month  | No         | Yes         |
| Replaces Mixpanel and Amplitude | No         | Yes         |
| Built-in A/B testing            | No         | Yes         |
| Mobile app recording            | No         | Yes         |
| Single source of truth          | No         | Yes         |

**Headline options:**
- Find out why xx% of Y Combinator startups choose PostHog
- The cheaper, all-in-one alternative to Hotjar 

## Mixpanel

|                       | **Mixpanel** | **PostHog** |
|-------------------------------|--------------|-------------|
| 1m free events every month    | No           | Yes         |
| A/B testing and feature flags | No           | Yes         |
| Run NPS and user surveys      | No           | Yes         |
| Totally transparent pricing   | No           | Yes         |
| Replace many tools with one   | No           | Yes         |

**Headline options:**
- xx% of Y Combinator startups that choose PostHog over Mixpanel
- “From customer support to content writers; engineers to marketers, everyone uses PostHog. We do everything from debugging support issues to measuring site traffic in PostHog.” – AssemblyAI
- "PostHog is our favorite tool; it’s the single source of truth for us." – Webshare

## Amplitude 

|                          | **Amplitude** | **PostHog** |
|-----------------------------------|---------------|-------------|
| 1 million free events every month | No            | Yes         |
| 5,000 free recordings every month | No            | Yes         |
| Totally transparent pricing       | No            | Yes         |
| No minimum annual commitments     | No            | Yes         |
| Replace all your tools with one   | No            | Yes         |

**Headline options:**
- xx% of Y Combinator startups choose PostHog over Amplitude
- The all-in-one alternative that won't break your wallet

## LaunchDarkly

|                               | **LaunchDarkly** | **PostHog** |
|------------------------------------|------------------|-------------|
| Unlimited seats                    | No               | Yes         |
| Simple, usage-based pricing        | No               | Yes         |
| Built-in product analytics         | No               | Yes         |
| Session replay, heatmaps & surveys | No               | Yes         |
| Single source of truth             | No               | Yes         |

**Headline options:**
- xx% of Y Combinator startups choose PostHog

**Comment by andyvan-ph** [comment]
Tagging @lottiecoxon @corywatilo for thoughts on art / mock-ups.

Tagging @ivanagas @charlescook-ph @joethreepwood @Lior539 @jtemperton for thoughts on copy / feedback on everything.

**Comment by charlescook-ph** [comment]
I think going for something where the competitor is all 'No' and we are all 'Yes' is a bit BS-y. Also the quotes could be more fun if they are Twitter ones that literally compare us to those competitors (even if it is a random person) - they usually have a bit more personality?

**Comment by andyvan-ph** [comment]
I'm not massively attached to the quotes tbh, but I think this is worth trying. 

It's an easy to understand format where we can explain very clearly how we're different. I think viewers are adult enough to accept this isn't an in-depth comparison, so long as the things we're saying are actually true, which they are.

One alternative I considered was something along the lines of "Reasons people switched to PostHog" – similar sort of concept but without the No / Yes dynamic.

**Comment by corywatilo** [comment]
Would Hey be designing these or do you want these done in-house? (Totally fine letting them run with it and critiquing from there, if so.)

**Comment by ivanagas** [comment]
Instead of us having a check and them having an X for all of the points, what if we both had checks for 4/5 points and we use the last one/two as the big differentiator(s) like low pricings, free tier, built for engineers, etc. This would play better to our strengths maybe.

**Comment by andyvan-ph** [comment]
> Would Hey be designing these or do you want these done in-house? (Totally fine letting them run with it and critiquing from there, if so.)

I figured you and/or Lottie might want first dibs, but this works for me.

**Comment by andyvan-ph** [comment]
> Instead of us having a check and them having an X for all of the points, what if we both had checks for 4/5 points and we use the last one/two as the big differentiator(s) like all-in-one, low pricings, free tier, built for engineers, etc. This would play better to our strengths while being less "BS-y".

I'll play around with this and report back.

**Comment by lottiecoxon** [comment]
![hptjar vs concept](https://github.com/user-attachments/assets/30773a35-ed86-442e-8790-362e076fa6bf)

**Comment by andyvan-ph** [comment]
Love this approach. Some notes:

- I think we'd need to tweak the scaling / size of the text on the checklist. I just capped what it looked like on my phone (see attached), and I think the main points are too small to be easily legible on a mobile screen.

- Would be good to play around with a bolder background colors than our standard light mode, or even a dark mode version. 

![IMG_4610](https://github.com/user-attachments/assets/8792fbab-c482-4007-9576-db4f33dc6dce)

**Comment by andyvan-ph** [comment]
@ivanagas @charlescook-ph – your takes on these alternatives:

|                                    | Hotjar | PostHog |
|------------------------------------|--------|---------|
| Session replay                     | Yes    | Yes     |
| Surveys & user feedback            | Yes    | Yes     |
| Heatmaps and scrollmaps            | Yes    | Yes     |
| Build by and for engineers         | No     | Yes     |
| Replaces Mixpanel, Amplitude & GA4 | No     | Yes     |

|                                    | Mixpanel | PostHog |
|------------------------------------|----------|---------|
| Product analytics                  | Yes      | Yes     |
| Dashboards and notebooks           | Yes      | Yes     |
| Feature flags and A/B testing      | No       | Yes     |
| 100% transparent and cheap pricing | No       | Yes     |
| Replace multiple tools with one    | No       | Yes     |

|                               | Amplitude | PostHog |
|-------------------------------|-----------|---------|
| Product analytics             | Yes       | Yes     |
| Session replays               | Yes       | Yes     |
| Feature flags and A/B testing | Yes       | Yes     |
| Generous free tier            | No        | Yes     |
| No minimum annual commitment | No        | Yes     |


|                             | LaunchDarkly | PostHog |
|-----------------------------|--------------|---------|
| Feature flags               | Yes          | Yes     |
| A/B testing                 | Yes          | Yes     |
| Built-in analytics          | No           | Yes     |
| Simple, transparent pricing | No           | Yes     |
| Single source of truth      | No           | Yes     |

**Comment by andyvan-ph** [comment]
Another revision @charlescook-ph @ivanagas.

FYI, I think we'll do a specific targeting on the ones that lead with YC stat to target smaller / younger companies on LinkedIn. Hotjar targeting can be wider. 

### The cheaper, all-in-one alternative to Hotjar

|                                    | Hotjar | PostHog |
|------------------------------------|--------|---------|
| Session replay and heatmaps                  | Yes    | Yes     |
| Surveys & user feedback            | Yes    | Yes     |
| 5k free replays every month           | No    | Yes     |
| Bulit-in A/B testing         | No     | Yes     |
| Replaces Mixpanel, Amplitude & GA4 | No     | Yes     |

Combined replay and heatmaps so we can include the 5k free replays.

### xx% of Y Combinator startups choose PostHog over Mixpanel

|                                    | Mixpanel | PostHog |
|------------------------------------|----------|---------|
| Product analytics                  | Yes      | Yes     |
| Dashboards and notebooks           | Yes      | Yes     |
| Feature flags and A/B testing      | No       | Yes     |
| 100% transparent and cheap pricing | No       | Yes     |
| Replace multiple tools with one    | No       | Yes     |

No change. I think this works as is.

### xx% of Y Combinator startups choose PostHog over Amplitude

|                               | Amplitude | PostHog |
|-------------------------------|-----------|---------|
| Product analytics             | Yes       | Yes     |
| Behaviorial cohorts              | Yes       | Yes     |
| Custom events & formulas | Yes       | Yes     |
| 100% transparent and cheap pricing           | No        | Yes     |
| No minimum annual commitment | No        | Yes     |

### xx% of Y Combinator startups choose PostHog over Amplitude

|                             | LaunchDarkly | PostHog |
|-----------------------------|--------------|---------|
| Feature flags               | Yes          | Yes     |
| A/B testing                 | Yes          | Yes     |
| Built-in analytics          | No           | Yes     |
| Simple, transparent pricing | No           | Yes     |
| Replaces Mixpanel, Amplitude & GA4      | No           | Yes     |

**Comment by andyvan-ph** [comment]
> I tried making the notebook bigger so the writing is more legible + experimented with colours, thoughts?

These look great @lottiecoxon. We should add our logo somewhere, but this concept looks good to me. Thoughts @corywatilo?

**Comment by andyvan-ph** [comment]
@lottiecoxon @corywatilo did you guys get any further with these?

**Comment by lottiecoxon** [comment]
developed to this point now- waiting on feedback from Cory
![hvsp 10](https://github.com/user-attachments/assets/c628616e-0653-4e98-b0e7-e8dda9298368)

**Comment by lottiecoxon** [comment]
Developed it to this point- awaiting feedback from Cory in Figma 

![hvsp 10](https://github.com/user-attachments/assets/032336a4-9064-4537-84a7-7189f8f301e0)

**Comment by andyvan-ph** [comment]
@lottiecoxon this looks good to me. One small note:

- We need the product names as well as / instead of the logos on the checklist. I like the arrow pointing bit, but it makes less sense if our name doesn't appear there.

**Comment by lottiecoxon** [comment]
been back and forth with Cory alot on this one - so have changed a bunch of things - hope this works 

![hvsp 16](https://github.com/user-attachments/assets/6e0b5409-c568-4123-9fb8-76e4176bf484)

**Comment by andyvan-ph** [comment]
Honestly, this feels like a step backwards. Angle on the comparison is jarring / hard to read, and feels like we're trying to cram in too many words.

In terms of clarity of messaging, Lottie's original concept is still the strongest. I liked the second quite a bit. A few tweaks to that could definitely work.

**Comment by lottiecoxon** [comment]
okay post insightful call with Andy I have reworked old design 

<img width="1200" alt="hvsp 17" src="https://github.com/user-attachments/assets/e12c1bfe-8c2c-4e6c-85ae-67bb13f88275">

**Comment by andyvan-ph** [comment]
Maybe shift the Hotjar column to the left a little so it appears centered under 'Hotjar'?

**Comment by lottiecoxon** [comment]
<img width="1200" alt="hvsp 18" src="https://github.com/user-attachments/assets/51d2203a-3a64-4507-9f79-930f7d735d7c">
<img width="1200" alt="hvsp 17" src="https://github.com/user-attachments/assets/f6cd2d0c-d098-4627-ab83-cfd1bdba3c97">

**Comment by andyvan-ph** [comment]
Ok, can we try a version where all the ones where Hotjar has 'x' are highlighted – i.e. highlight the whole item, not just specific words.

**Comment by lottiecoxon** [comment]
<img width="1200" alt="hvsp 18" src="https://github.com/user-attachments/assets/ce55d206-aeef-46d5-9a38-8f34b86c3732">

**Comment by andyvan-ph** [comment]
@lottiecoxon what do you think about the highlighting? I think it works, but open to dissenting opinions.

**Comment by lottiecoxon** [comment]
it draws the eye onto the most important parts - I'd say yes to the one above

**Comment by andyvan-ph** [comment]
sorry to be tedious, but could we try one more version where that is underlined instead of highlighted?

**Comment by lottiecoxon** [comment]
the title or the text or both?

**Comment by lottiecoxon** [comment]
<img width="1200" alt="hvsp 19" src="https://github.com/user-attachments/assets/92726f97-90e5-4b59-846a-ef9590814a70">

**Comment by lottiecoxon** [comment]
<img width="1200" alt="hvsp 20" src="https://github.com/user-attachments/assets/f4e6687c-2afd-49e6-884c-442ed728d2d3">

**Comment by andyvan-ph** [comment]
Lovely, thanks

**Comment by corywatilo** [comment]
What happened to the drafts we were working on that did a rev on this style?

**Comment by lottiecoxon** [comment]
I showed Andy -but he felt that for what he was after was more similar to an earlier version of the design hence the roll back

**Comment by andyvan-ph** [comment]
***Bold text** denotes highlighted text in headline

The **cheaper**, all-in-one **alternative to Hotjar**

|                                    | Hotjar | PostHog |
|------------------------------------|--------|---------|
| Session replay and heatmaps                  | Yes    | Yes     |
| Surveys & user feedback            | Yes    | Yes     |
| 5k free replays every month           | No    | Yes     |
| Bulit-in A/B testing         | No     | Yes     |
| Replaces Mixpanel, Amplitude & GA4 | No     | Yes     |

### The **cheaper**, all-in-one **alternative to Mixpanel**

|                                    | Mixpanel | PostHog |
|------------------------------------|----------|---------|
| Product and web analytics             | Yes      | Yes     |
| Dashboards and notebooks           | Yes      | Yes     |
| Feature flags and Experiments      | No       | Yes     |
| User surveys & feedback | No       | Yes     |
| Replaces Hotjar, GA4 and LaunchDarkly    | No       | Yes     |

### xx% of Y Combinator startups choose PostHog over Amplitude

|                               | Amplitude | PostHog |
|-------------------------------|-----------|---------|
| Product analytics             | Yes       | Yes     |
| Session replay              | Yes       | Yes     |
| User surveys and feedback | No       | Yes     |
| Generous free tier           | No        | Yes     |
| No minimum annual commitment | No        | Yes     |

### The **cheaper**, all-in-one **alternative to LaunchDarkly**

|                             | LaunchDarkly | PostHog |
|-----------------------------|--------------|---------|
| Feature flags               | Yes          | Yes     |
| Experiments               | Yes          | Yes     |
| Product and web analytics          | No           | Yes     |
| Session replay and heatmaps | No           | Yes     |
| Replaces Mixpanel, Amplitude & GA4      | No           | Yes     |

**Comment by lottiecoxon** [comment]
As requested- but ran into some issues:
- are we highlighting all the sections that our opposition has an X next to, if so this can get a bit noisy 
- colours for backgrounds should vary but unsure which colours to go for as the colours on the website align with specific products but with these comparison ads feature multiple product mentions - so I'm stuck here @corywatilo any ideas?
- some of the ads are a bit wordy

![amplitude](https://github.com/user-attachments/assets/19f4e3c3-5e28-4d5e-aad8-fd68f75ad101)
![heap](https://github.com/user-attachments/assets/2aa52be2-fb82-4476-904f-f236078b475c)
![hotjar](https://github.com/user-attachments/assets/2cd4c180-6b9a-4a94-ab6d-
![mixpanel](https://github.com/user-attachments/assets/642898fc-293a-4692-8344-8798dccf2a4b)
9f33d4cd7a72)
![launchdarkly](https://github.com/user-attachments/assets/d6dc311e-fc49-4e4d-a26f-2d6b680ad092)

**Comment by corywatilo** [comment]
I left some comments [on this artboard](https://www.figma.com/design/QCYxjm4wVVVvs6UA27xRmz/Website-art?node-id=2514-219&t=t8flfcFcpbVr70WE-1)

For background colors, it should match whichever product we're highlighting. Since LD is known for FF, use the green we use there. If we're talking about a product analytics competitor, we'd want to use blue.

**Comment by lottiecoxon** [comment]
![amplitude](https://github.com/user-attachments/assets/5a471e90-d4c5-4e42-a44b-a5817ce8c39a)
![heap](https://github.com/user-attachments/assets/13ab2594-629e-4493-b679-4e3e9a5777ec)
![launchdarkly](https://github.com/user-attachments/assets/72f66d41-da12-4e68-8f5f-c7697eded3f0)
![mixpanel](https://github.com/user-attachments/assets/dc651ea4-b4e5-4821-a286-e67cbb2d1115)
![hotjar](https://github.com/user-attachments/assets/c62628f5-6439-44e8-8e30-a02b4e395bb9)

**Comment by rafaeelaudibert** [comment]
This is from 2024, maybe we should close this now?
